README.md

```diff
@@ -219,80 +219,58 @@ The corresponding function in on Tarantool Cartridge router may look like (uses
 
 See more examples in the module tests.
 
-#### Call stored functions in Tarantool instance
-
-You can bind repository methods to calls of the stored functions in the Tarantool instance using the `@Query` annotation
-with the stored function name specified in the `functionName` parameter.
-For such methods, you can specify the stored function response format so that it will be parsed correctly. The response
-format may be either an object (and a list of objects) or a tuple (and a list of tuples).
-
-```java
-@Data
-@Tuple
-public class SampleUser {
-    private String name;
-    private String lastName;
-}
-```
+##### Specify output
+For such methods, you can specify the stored function response format so that it will be parsed correctly.
+The response format is also specified in the `@Query` annotation in the output parameter.
 
 ```java
 public interface SampleUserRepository extends TarantoolRepository<SampleUser, String> {
-    @Query(function = "returning_sample_user_object")
-    SampleUser returningSampleUserObject(String name);
+    @Query(function = "get_users_with_age_gt", output = TarantoolSerializationType.TUPLE)
+    List<SampleUser> usersWithAgeGreaterThen(Integer age);
 
-    @Query(function = "get_predefined_users")
-    List<SampleUser> getPredefinedUsers();
+    @Query(function = "get_predefined_user", output = TarantoolSerializationType.AUTO)
+    SampleUser predefinedUser();
 }
 ```
 
-```java
-@Tuple
-public class Book {
-  @Id
-  private Integer id;
-  private String name;
-}
+```lua
+function get_users_with_gt_age(age)
+    return crud.select("sample_user", { { ">", "age", age } })
+end
 
-// Here, the space name is "another_space" for all @Query methods
-@Tuple("another_space") 
-public interface BookRepository extends TarantoolRepository<Book, Integer> {
-    
-    // The space name here overrides "another_space" to "test_space" for the current method
-    @Tuple("test_space") 
-    @Query(function = "do_something")
-    Optional<Book> doSomething(Book book);
-}
+function get_predefined_user()
+    return { name = "John", age = 46 }
+end
 ```
+The first function returns users who are older than the specified age.
+In the Repository, we indicate that we expect `TUPLE`, because `crud.select` returns the result in compressed format, flatten tuples.
+The keys for mapping the result into a Java object are obtained by the connector by the class name in returnType or by the specified `@Tuple` above it.
 
-Here the space name is not specified neither in a `@Tuple` annotation on the repository interface, nor on the entity class, so it will be "book_entity" according to the entity class name, turned into "lower snake case".
-But for the "doSomething" method the space name will be overridden to "test_space".
-```java
-@Tuple
-public class BookEntity {
-  @Id
-  private Integer id;
-  private String name;
-}
+The second function returns a ready-made result with keys.
```

Comment:
It is better to put the text closer to examples. Otherwise the reader will have to jump back and forth to the description.

Reply:
Put description between java code and lua code?

Reply:
Nevermind, I took these changes wrong